### ISSUE TITLE
Lighthouse bug with Salesforce ISML code

### ISSUE THREAD

**Issue by jotero3mt** [GoogleChrome/web.dev#8625]
**Describe the bug**
Lighthouse flagged lists in the menu that contain content elements that shouldn't be in a list.
The problem is that our lists only have <li> elements. We think Lighthouse is reading isml code as html before it's rendered so it throws this error when the rendered html is correct.

**To Reproduce**
1. Go to DevTools
2. Go to a Salesforce site (like https://www.guess.com/us/en/home)
3. Launch Lighthouse -> Mobile device
4. Go to Lighthouse -> Accessibility
5. Check errors reported

**Expected behavior**
Lighthouse analyzes the structure once the code is rendered and it reports correctly if there are elements that should not belong to a list. But it does not report false positives.

**Screenshots**
This is the problem reported by Lighthouse, the link clicked is underlined in red:

<img width="727" alt="GLCG-2671 - 01 Problem reported" src="https://user-images.githubusercontent.com/91899875/187850642-2659ef13-07c2-4b59-bcc7-4604725ee07a.png">

This is the line where the link sends me to, as you can see, there's nothing but <li> elements inside the <ul> element:

<img width="740" alt="GLCG-2671 - 02 Elements loaded" src="https://user-images.githubusercontent.com/91899875/187850671-b9ce057c-1a84-4b43-9540-5df4641e66dd.png">

This is taken from the template that renders this element:

<img width="1071" alt="GLCG-2671 - 03 Code template" src="https://user-images.githubusercontent.com/91899875/187850707-0e78e435-8dcf-4735-8db5-c20f6aaa5c28.png">

Isif and isloop elements are isml elements used to render the template, those can not be removed since it's programming logic, they are necessary.

These are other two errors reported by Lighthouse:

<img width="737" alt="GLCG-2671 - 04 Problem reported 2" src="https://user-images.githubusercontent.com/91899875/187850727-6f73ad0c-51a8-48fd-b93c-8c26167ea030.png">

It clearly says that those li elements are not contained in an ul element, but...

<img width="737" alt="GLCG-2671 - 05 Elements loaded 2" src="https://user-images.githubusercontent.com/91899875/187850749-30539105-5cbc-4f65-81d5-bd1400e513c6.png">

Once we click, it takes us to this element, and we can clearly see it's contained inside a ul element.

<img width="748" alt="GLCG-2671 - 06 Code template 2" src="https://user-images.githubusercontent.com/91899875/187850765-6cb2209a-d32d-47bc-ba2b-64da19d010cb.png">

We can see it's exactly the same in the template that is rendered.

In conclusion: I think Lighthouse is reporting errors and trying to mark isml elements as html elements before everything is rendered. I think we should ignore these reports as long as we don't see the error represented when we click on those links.
